lib/ruby_indexer/test/rbs_indexer_test.rb

```diff
@@ -74,5 +74,229 @@ def test_location_and_name_location_are_the_same
 
       assert_same(entry.location, entry.name_location)
     end
+
+    def test_rbs_method_with_required_positionals
+      entries = @index["crypt"] # https://www.rubydoc.info/stdlib/core/String#crypt-instance_method
+      assert_equal(1, entries.length)
+
+      entry = entries.first
+      signatures = entry.signatures
+      assert_equal(1, signatures.length)
+
+      first_signature = signatures.first
+
+      # (::string salt_str) -> ::String
```

Comment:
I copied in the RBS definitions for easy reference, but we could remove them.

Reply:
I like those, it's easier to understand 👍